democracy/views/comment.py

```diff
@@ -60,13 +60,13 @@ def _check_may_comment(self, request):
         parent = self.get_comment_parent()
         try:
             may_comment = parent.may_comment(request)
+            if not may_comment:  # pragma: no cover
```

Comment:
Technically yes, but it's a fallback case; right now `may_comment` implementations always return True or raise a ValidationError.